builder/vmware/common/step_register.go

```diff
@@ -52,35 +54,45 @@ func (s *StepRegister) Cleanup(state multistep.StateBag) {
 	_, cancelled := state.GetOk(multistep.StateCancelled)
 	_, halted := state.GetOk(multistep.StateHalted)
 	if (s.KeepRegistered) && (!cancelled && !halted) {
-		ui.Say("Keeping virtual machine registered with ESX host (keep_registered = true)")
+		log.Printf("Virtual machine will remain registered; `keep_registered` set to `true`.")
 		return
 	}
 
 	if remoteDriver, ok := driver.(RemoteDriver); ok {
 		if s.SkipExport && !cancelled && !halted {
-			ui.Say("Unregistering virtual machine...")
+			ui.Say("Removing virtual machine from inventory...")
 			if err := remoteDriver.Unregister(s.registeredPath); err != nil {
-				ui.Error(fmt.Sprintf("Error unregistering VM: %s", err))
+				errorMessage := fmt.Sprintf("error removing virtual machine: %s", err)
+				ui.Error(errorMessage)
+				log.Printf(errorMessage)
 			}
 
 			s.registeredPath = ""
 		} else {
-			ui.Say("Destroying virtual machine...")
+			ui.Say("Deleting virtual machine...")
 			if err := remoteDriver.Destroy(); err != nil {
-				ui.Error(fmt.Sprintf("Error destroying VM: %s", err))
+				errorMessage := fmt.Sprintf("error deleting virtual machine: %s", err)
+				ui.Error(errorMessage)
+				log.Printf(errorMessage)
```

Comment:
I would remove the `log.Printf` if there's already a `ui.Error`, as they'll be visible in any case

Reply:
Resolved in 13c8c54 .